webpack.config.js

```diff
@@ -98,19 +100,14 @@ module.exports = (env, argv) => {
             ]
         },
         plugins: [
-            new ModuleFederationPlugin({
+
+            new ModuleFederationPlugin(getModuleFederationConfig(packageJson, {
                 name: "jahiaUi",
                 library: { type: "assign", name: "appShell.remotes.jahiaUi" },
```

Comment:
Library is not necessary, it will be added by getModuleFederationConfig which will use the name property that you set